scripts/runner.go

```diff
@@ -0,0 +1,37 @@
+package scripts
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"runtime"
+)
+
+// A version of exec.LookPath that can find our scripts
+// It works by manipulating $PATH
+func LookPath(file string) (string, error) {
+	_, goFile, _, ok := runtime.Caller(0)
+	if !ok {
+		return "", errors.New("failed to find path of scripts via Go file name")
+	}
+	if err := pushToPath(path.Dir(goFile)); err != nil {
+		return "", err
+	}
+	return exec.LookPath(file)
```

Comment:
So if I got it right, first we manipulate the PATH to include the directory of this very script, then if everything went well we run `LookPath` which now search through a modified path? 
if so, might want to elaborate on this on the comment doc...

Reply:
done